packages/framework/src/Migrations/Version20241205144230.php

```diff
@@ -0,0 +1,43 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Shopsys\FrameworkBundle\Migrations;
+
+use Doctrine\DBAL\Schema\Schema;
+use Shopsys\MigrationBundle\Component\Doctrine\Migrations\AbstractMigration;
+
+class Version20241205144230 extends AbstractMigration
+{
+    /**
+     * @param \Doctrine\DBAL\Schema\Schema $schema
+     */
+    public function up(Schema $schema): void
+    {
+        $this->sql('CREATE EXTENSION IF NOT EXISTS pg_trgm');
+
+        //product indexes
+        $this->sql('CREATE INDEX products_catnum_trgm_idx ON products USING gin (normalized(catnum) gin_trgm_ops);');
```

Comment:
this should also be mentioned in the upgrade notes - developers should create such indexes for all the fields they are normalizing and using in their custom admin search queries, right?

Reply:
I would add this to the upgrade notes (under the mention of the migration):

```
- check your code for usages of the `NORMALIZED(columnName) LIKE NORMALIZED(:text)` and create the indexes for corresponding columns the same way they are created in the migration `Version20241205144230`